pkg/virt-controller/services/net_binding.go

```diff
@@ -46,31 +47,60 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 			}
 		}
 	}
+
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.Slirp != nil {
+			pluginSidecars = append(pluginSidecars, SlirpNetBindingPluginSidecar(vmi, config, recorder))
+			break
```

Comment:
I do not think this is needed. Embedding here the logic that there can be only one such interface is not a fit IMO.